quantile/stream.go

```diff
@@ -77,15 +77,18 @@ func NewHighBiased(epsilon float64) *Stream {
 // is guaranteed to be within (Quantile±Epsilon).
 //
 // See http://www.cs.rutgers.edu/~muthu/bquant.pdf for time, space, and error properties.
-func NewTargeted(targets map[float64]float64) *Stream {
+func NewTargeted(targetMap map[float64]float64) *Stream {
+	// Convert map to slice to avoid slow iteration on map
```

Comment:
Nit: End sentence with a period.

Reply:
And perhaps add a hint that the iteration in the anonymous function below is in the hot path (given that that's what I missed when I changed it in dab8af3af48f705feb0b7b683b385ffc11ccb3ca ).